src/components/maskedInput/index.tsx

```diff
@@ -1,11 +1,18 @@
-import React, {forwardRef} from 'react';
+import React, {forwardRef, useEffect} from 'react';
 // @ts-expect-error
 import MaskedInputOld from './old';
 import MaskedInputNew, {MaskedInputProps} from './new';
+import {LogService} from 'services';
 
 function MaskedInputMigrator(props: any, refToForward: any) {
   const {migrate, ...others} = props;
 
+  useEffect(() => {
+    if (!migrate) {
+      LogService.warn(`UILib MaskedInput implementation has been updated and now requires manual migration. To proceed, pass the "migrate" prop and check if the functionality works as expected.`);
```

Comment:
Do we have a migration guide to point users to?

Reply:
Nope, there's an API parity also there's not much usage of this component